utils/bazel/llvm-project-overlay/llvm/lit_test.bzl

```diff
@@ -35,7 +35,7 @@ def lit_test(
         args = args + ["-v"] + ["$(execpath %s)" % src for src in srcs],
         data = data + srcs,
         legacy_create_init = False,
-        deps = [Label("//llvm:lit")],
+        deps = [Label("//llvm:lit")] + kwargs.pop("deps", []),
```

Comment:
Instead of `pop()` here, can you add a `deps` param to the method and follow the pattern as for args/data?

```
    deps = deps or []

    native.py_test(
        ...
        deps = deps + [Label("//llvm:lit")],
    )
````

Reply:
Thanks you for the review, updated.